packages/editor/src/lib/hooks/usePassThroughWheelEvents.ts

```diff
@@ -1,16 +1,23 @@
 import { RefObject, useEffect } from 'react'
 import { preventDefault } from '../utils/dom'
 import { useContainer } from './useContainer'
+import { useEvent } from './useEvent'
 
 /** @public */
-export function usePassThroughWheelEvents(ref: RefObject<HTMLElement>) {
+export function usePassThroughWheelEvents(
+	ref: RefObject<HTMLElement>,
+	condition?: (e: WheelEvent) => boolean
```

Comment:
Not sure if condition is necessary here, we could always do this check. I can't think of anywhere scrollable that I'd want to pass through wheel events. (Could also be extracted out into a separate easy-to-land PR)